rcl/include/rcl/timer.h

```diff
@@ -148,6 +153,9 @@ rcl_timer_fini(rcl_timer_t * timer);
  * the timer's period has not yet elapsed.
  * It is up to the calling code to make sure the period has elapsed by first
  * calling rcl_timer_is_ready().
+ * If the callback option was set to null in init, no callback is fired.
```

Comment:
It might be better to phrase this as "If the callback is NULL (either set in init or exchanged for NULL later) ..." or something to that effect.